README.md

```diff
@@ -23,3 +24,41 @@
 
 
 # DroneVis: Full compatible drone library to automate computer vision algorithms on parrot drones.
+
+**DroneVis** is a cutting-edge drone software library that has been specifically designed for use with the AR. Drone 2.0. It has been extensively tested both indoors and outdoors, and offers a wide range of features including adaptability in connecting to the drone, advanced computer vision algorithms, and a user-friendly interface. This makes it easy for users to take full advantage of the drone's capabilities and control it with simple commands.All of the implemented real-time data, inference, and detection achieve a minimum ``fps >= 4.5`` on an Intel core 8 CPU.
+
+## Features
+- Unified state-of-the art computer vision algoritms
+- Full control over the drone
+- PEP8 compliant (unified code style)
+- Documented functions and classes
+- Tests, high code coverage and type hints
+- Clean code
+- Multiple implementations for the same models
+- Two UI for easier usage (GUI, CLI)
+
+
+
+| Drone Control         | Computer Vision Models| Usage                 |   Implementation      |
```

Comment:
The table is a bit confusing to me as I am trying to read one row at a time. 
Is the Drone control column separated from the rest of the table.

Reply:
Maybe we should discuss a better alternative to represent the features in the next meeting.